.github/workflows/build.yml

```diff
@@ -461,9 +488,19 @@ jobs:
           # the caching step and not wiping userdata on boot.
           emulator-options: -no-snapshot -no-window -noaudio -no-boot-anim -camera-back none
           disable-animations: true
-          script: |
-            $ANDROID_SDK_ROOT/platform-tools/adb forward tcp:5432 tcp:5432
-            $ANDROID_SDK_ROOT/platform-tools/adb push ${{ github.workspace }}/BinaryCache/ds2/ds2 /data/local/tmp
-            $ANDROID_SDK_ROOT/platform-tools/adb shell "chmod +x /data/local/tmp/ds2"
-            $ANDROID_SDK_ROOT/platform-tools/adb shell "/data/local/tmp/ds2 platform --server --daemonize --listen localhost:5432"
-            ${{ github.workspace }}/BinaryCache/llvm/bin/lldb-dotest --out-of-tree-debugserver --arch ${{ matrix.processor}} --platform-name remote-android --platform-url connect://localhost:5432 --platform-working-dir /data/local/tmp --compiler=${{ steps.setup-ndk.outputs.ndk-path }}/toolchains/llvm/prebuilt/linux-x86_64/bin/clang --excluded ${{ github.workspace }}/SourceCache/ds2/Support/Testing/Excluded/ds2/android-x86_64.excluded --excluded ${{ github.workspace }}/SourceCache/ds2/Support/Testing/Excluded/upstream/non-debugserver-tests.excluded --excluded ${{ github.workspace }}/SourceCache/ds2/Support/Testing/Excluded/upstream/android.excluded
+          script: "\
```

Comment:
I tried a variation of `>` but it squashed it all onto one line, but I realize I never tried `>` with `\n` to force the newlines so that might work. Will try and see.

Reply:
According to what I've read, this syntax should work (double newlines -> single newline) but it ends up with no newlines:

```
          script: >
            $ANDROID_SDK_ROOT/platform-tools/adb forward tcp:5432 tcp:5432

            $ANDROID_SDK_ROOT/platform-tools/adb push "${{ github.workspace }}/BinaryCache/ds2/ds2" /data/local/tmp

            $ANDROID_SDK_ROOT/platform-tools/adb shell "chmod +x /data/local/tmp/ds2"

            $ANDROID_SDK_ROOT/platform-tools/adb shell "/data/local/tmp/ds2 platform --server --daemonize --listen localhost:5432"

            ${{ github.workspace }}/BinaryCache/llvm/bin/lldb-dotest
              --out-of-tree-debugserver
              --arch ${{ matrix.processor}}
              --platform-name remote-android
              --platform-url connect://localhost:5432
              --platform-working-dir /data/local/tmp
              --compiler=${{ steps.setup-ndk.outputs.ndk-path }}/toolchains/llvm/prebuilt/linux-x86_64/bin/clang
              --excluded ${{ github.workspace }}/SourceCache/ds2/Support/Testing/Excluded/ds2/android-${{ matrix.processor }}.excluded
              --excluded ${{ github.workspace }}/SourceCache/ds2/Support/Testing/Excluded/upstream/non-debugserver-tests.excluded
              --excluded ${{ github.workspace }}/SourceCache/ds2/Support/Testing/Excluded/upstream/android.excluded
              ${{ matrix.test-subdirs }
```